src/style.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: 'Avenirbook';
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Avenirheavy';
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  font-weight: 300;
+
+}
+.search-bar {
+  width: 100%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  height: 70px;
+  border-radius: 4px;
+  background-color: #FFFFFF;
+  border: 1px solid #E1E7ED;
+  box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1);
+  background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: left;
+  background-position-x: 26px;
+  background-size: 19px 19px;
+  text-indent: 62px;
+
```

Comment:
There is an extra blank line, consider removing it to make the code more consistent 🧹